examples/ui/scrollbars.rs

```diff
@@ -0,0 +1,201 @@
+                        Spawn(text_row("Zeta Function")),
+                        Spawn(text_row("Lambda Calculus")),
+                        Spawn(text_row("Nu Metal")),
+                        Spawn(text_row("Pi Day")),
+                        Spawn(text_row("Chi Pants")),
+                        Spawn(text_row("Psi Powers")),
+                        Spawn(text_row("Omega Fatty Acid")),
+                    )),
+                ))
+                .id();
+
+            // Vertical scrollbar
+            parent.spawn((
+                Node {
+                    min_width: Val::Px(8.0),
+                    grid_row: GridPlacement::start(1),
+                    grid_column: GridPlacement::start(2),
+                    ..default()
+                },
+                CoreScrollbar {
+                    orientation: ControlOrientation::Vertical,
+                    target: scroll_area_id,
+                    min_thumb_size: 8.0,
+                },
+                Children::spawn(Spawn((
+                    Node {
+                        position_type: PositionType::Absolute,
+                        ..default()
+                    },
+                    Hovered::default(),
+                    BackgroundColor(colors::GRAY2.into()),
+                    BorderRadius::all(Val::Px(4.0)),
+                    CoreScrollbarThumb,
+                ))),
+            ));
+
+            // Horizontal scrollbar
+            parent.spawn((
+                Node {
+                    min_height: Val::Px(8.0),
+                    grid_row: GridPlacement::start(2),
+                    grid_column: GridPlacement::start(1),
+                    ..default()
+                },
+                CoreScrollbar {
+                    orientation: ControlOrientation::Horizontal,
+                    target: scroll_area_id,
+                    min_thumb_size: 8.0,
+                },
+                Children::spawn(Spawn((
+                    Node {
+                        position_type: PositionType::Absolute,
+                        ..default()
+                    },
+                    Hovered::default(),
+                    BackgroundColor(colors::GRAY2.into()),
+                    BorderRadius::all(Val::Px(4.0)),
+                    CoreScrollbarThumb,
+                ))),
+            ));
+        }),)),
+    )
+}
+
+/// Create a list row
+fn text_row(caption: &str) -> impl Bundle {
+    (
+        Text::new(caption),
+        TextFont {
+            font_size: 14.0,
+            ..default()
+        },
+    )
+}
+
+// Update the color of the scrollbar thumb.
+fn update_scrollbar_thumb(
+    mut q_thumb: Query<
+        (&mut BackgroundColor, &Hovered),
+        (With<CoreScrollbarThumb>, Changed<Hovered>),
+    >,
+) {
+    for (mut thumb_bg, Hovered(is_hovering)) in q_thumb.iter_mut() {
+        let color: Color = if *is_hovering {
+            // If hovering, use a lighter color
+            colors::GRAY3
```

Comment:
It should also use the lighter colour if the pointer unhovers while dragging.

Reply:
I agree; I just need to figure out how to do it. Unlike the slider where the `Hovered` component is on the entire slider, in this case the `Hovered` component is on the thumb. This means that the two input signals are coming from two different entities, which means that simple change detection query filters won't work.

Reply:
I suppose I could put a `Pressed` marker on the thumb while a drag is active. But then I have to deal with the pain of `RemovedComponents`, yuck.

Reply:
OK I believe I have solved the problem: I moved the drag state to the thumb entity, allowing a single query filter for both hover and drag. (Yet another way in which scrollbars diverge from sliders).